src/main/kotlin/io/cloudflight/cleancode/archunit/rules/jpa/KotlinEntityNullableCondition.kt

```diff
@@ -63,6 +63,7 @@ internal class KotlinEntityNullableCondition :
         findAnnotation<Id>(memberProperty) { false }?.let { nullable = it }
         findAnnotation<GeneratedValue>(memberProperty) { nullableIfNotTypeLong(memberProperty) }?.let { nullable = it }
         findAnnotation<ManyToOne>(memberProperty) { manyToOne -> manyToOne.optional }?.let { nullable = it }
+        findAnnotation<OneToOne>(memberProperty) { manyToOne -> manyToOne.optional }?.let { nullable = it }
```

Comment:
Please rename the lambda variable to reflect the correct annotation
```suggestion
        findAnnotation<OneToOne>(memberProperty) { oneToOne -> oneToOne .optional }?.let { nullable = it }
```